fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/temp/Type.java

```diff
@@ -290,7 +290,6 @@ static String uniqueCompliantTypeName() {
         INT(Integer.class, FieldDescriptorProto.Type.TYPE_INT32, true, true),
         LONG(Long.class, FieldDescriptorProto.Type.TYPE_INT64, true, true),
         STRING(String.class, FieldDescriptorProto.Type.TYPE_STRING, true, false),
-        TUPLE(List.class, null, false, false),
         RECORD(Message.class, null, false, false),
```

Comment:
With the unification of tuples and records, are field _indexes_ still optional?

Reply:
I think this was resolved by discussion with @normen662.